CMakeLists.txt

```diff
@@ -5,22 +5,16 @@ enable_testing()
 
 set(CMAKE_OSX_DEPLOYMENT_TARGET "10.12" CACHE STRING "Minimum OS X deployment version")
 
-project(VULKAN_PROFILES LANGUAGES CXX C)
+project(VULKAN_PROFILES LANGUAGES CXX)
 
 # Regenerating profiles requires Python 3.7.2 or better (for typing.OrderedDict).
 # Regeneration jsoncpp requires Python 3.
 # Otherwise Python is not required, as checked-in generated code and
 # headers will be used.
 option(PROFILES_BUILD_TESTS "Build profile tests" ON)
 
-set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/lib" CACHE PATH "Archive output dir.")
-set(CMAKE_LIBRARY_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/lib" CACHE PATH "Library output dir.")
-set(CMAKE_PDB_OUTPUT_DIRECTORY     "${CMAKE_BINARY_DIR}/bin" CACHE PATH "PDB (MSVC debug symbol)output dir.")
-set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin" CACHE PATH "Executable/dll output dir.")
```

Comment:
This was needed for the jsoncpp but isn't actually good practice. Now that jsoncpp is built standalone and consumed via find_package we don't need to do this anymore.

Reply:
This is one of the benefits of using find_package versus add_subdirectory.

add_subdirectory makes you have to care about the quality of the CMake code of the libraries you consume. Versus find_package just uses the already built artifacts.

Reply:
Looks like profiles maybe depending on the layout of the build directory so I reverted this change for now.